cli/analytics.go

```diff
@@ -0,0 +1,78 @@
+package cli
+
+import (
+	"encoding/json"
+	"os"
+
+	"github.com/bitrise-io/bitrise-plugins-analytics/analytics"
+	"github.com/bitrise-io/bitrise-plugins-analytics/configs"
+	"github.com/bitrise-io/bitrise/models"
+	"github.com/bitrise-io/bitrise/plugins"
+	log "github.com/bitrise-io/go-utils/log"
+	"github.com/hashicorp/go-version"
+	"github.com/pkg/errors"
+)
+
+func ensureFormatVersion(pluginFormatVersionStr, hostBitriseFormatVersionStr string) (string, error) {
+	if hostBitriseFormatVersionStr == "" {
+		return "This analytics plugin version would need bitrise-cli version >= 1.6.0 to submit analytics", nil
+	}
+
+	hostBitriseFormatVersion, err := version.NewVersion(hostBitriseFormatVersionStr)
+	if err != nil {
+		return "", errors.Wrapf(err, "Failed to parse bitrise format version (%s)", hostBitriseFormatVersionStr)
+	}
+
+	pluginFormatVersion, err := version.NewVersion(pluginFormatVersionStr)
+	if err != nil {
+		return "", errors.Errorf("Failed to parse analytics plugin format version (%s), error: %s", pluginFormatVersionStr, err)
+	}
+
+	if pluginFormatVersion.LessThan(hostBitriseFormatVersion) {
```

Comment:
NOTE/Q: is this check block still required?

Reply:
for now yes: in this PR, I just prepare for the change